### ISSUE TITLE
Allow empty password

### ISSUE THREAD

**Issue by teh-mICON** [restic/restic#4137]
I've been searching for a backup solution for a while now. I want to run a script that backs up my directory incrementally to a remote server where snapshots are being kept.

restic seems to be this tool but when I started using it, I noticed that I am required to use a password.

I do NOT like this. The data I'm backing up is to me worth a LOT more than it being secure from outside forces.

Being forced to remember or store a password is just a security risk for my files in the sense that I might actually lose them which is the opposite of what I'm trying to achieve.

To sum it up: I would really like to use restic but not if I'm forced to keep a password. I do not want to scramble for this password in a couple of years in the high stress event of actually needing a backup.

So my feature request here is to allow an empty password.
Alternatively I would welcome recommendations for a different tool that does what restic does but without the absolute need to password encrypt.

**Comment by rawtaz** [comment]
This is a duplicate of #1786, so closing for that reason.
